index.html

```diff
@@ -0,0 +1,286 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Development course</title>
+</head>
+
+<body>
+  <div class="mobile-logo-wrap">
+    <div>
+      <a href="index.html">
+        <img src="Images/main-logo.png" alt="developer week global">
+      </a>
+    </div>
+  </div>
+
+  <div class="mobile-menu-wrap">
+    <div class="mobile-menu-button">
+      <a href="#" id="mobile-menu-button">
+        <img src="Images/mobile_menu.png" alt="Mobile menu button">
+      </a>
+    </div>
+
+    <div class="mobile-menu-display display-none" id="mobile-menu-display">
+      <div class="mobile-menu-item align-self-fx-end">
+        <button id="close-mobile-menu">&#10799;</button>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>News</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Sponsor</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Join</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="#">
+          <p>Program</p>
+        </a>
+      </div>
+
+      <div class="mobile-menu-item">
+        <a href="about.html">
+          <p>About</p>
+        </a>
+      </div>
+    </div>
+  </div>
+  <!-- Desktop navbar  -->
+  <div class="wrap header-wrap">
+    <div class="main-menu">
+
+      <div class="header-content">
+        <div class="main-logo">
+          <a href="index.html">
+            <img src="Images/main-logo.png" alt="developer week global">
+          </a>
+        </div>
+        <div class="menus">
+          <ul>
+            <li class="menu-item"><a href="about.html">About</a></li>
+            <li class="menu-item"><a href="#">Program</a></li>
+            <li class="menu-item"><a href="#">Join</a></li>
+            <li class="menu-item"><a href="#">Sponsor</a></li>
+            <li class="menu-item"><a href="#">News</a></li>
+          </ul>
+        </div>
+      </div>
+
+    </div>
+  </div>
+
+  <div class="main-wrap wrap">
+    <section>
+      <img id='title-image' src="Images/developer-week-logo.png" alt="developer week february 15-17 sf bay area february 21-23">
```

Comment:
- [ ] Kindly ensure that your headline section title matches the design template in terms of color and font size as it is important to stick to the design as much as possible 👍🏽

| Design Template  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/197051244-581e0a61-1abd-4f41-b94d-93a2f2d3c33e.png)  | ![image](https://user-images.githubusercontent.com/63473584/197051549-17c7be56-06eb-4d13-b325-8f8ae5d22bba.png)  |